core/src/test/kotlin/org/stellar/anchor/api/sep/sep10/ChallengeRequestTest.kt

```diff
@@ -17,12 +17,12 @@ internal class ChallengeRequestTest {
       ChallengeRequest.builder()
         .account(TEST_ACCOUNT)
         .memo(TEST_MEMO)
-        .homeDomain(TEST_HOME_DOMAIN)
+        .homeDomain(TEST_WEB_AUTH_DOMAIN)
```

Comment:
We should keep the name, since `ChallengeRequest.homeDomain` should be equal to the domain hosting a `stellar.toml` file, not the domain hosting the SEP-10 server.

Reply:
👍 Fixed.